classes/Product.php

```diff
@@ -4067,6 +4068,9 @@ public static function getPriceFromOrder(
         if (Combination::isFeatureActive()) {
             $sql->where('od.`product_attribute_id` = ' . $combinationId);
         }
+        if (Customization::isFeatureActive()) {
+            $sql->where('od.`id_customization` = ' . $ustomizationId);
```

Comment:
Missing the "c" and (int)

Reply:
You're right, it's the rush!
Thank you.

Reply:
I didn't insert the cast to "int" to maintain the behavior that exists in the other instructions, then there is int as the type in the parameter, so it should be enough.